lib/gems/pending/util/MiqSshUtil.rb

```diff
@@ -149,18 +139,18 @@ def exec(cmd, doneStr = nil, stdin = nil)
 
         channel.on_close do |_channel|
           $log.debug "MiqSshUtil::exec - Command: #{cmd}, exit status: #{status}" if $log
-          unless signal.nil? || status.zero?
-            raise "MiqSshUtil::exec - Command #{cmd}, exited with signal #{signal}" unless signal.nil?
-            raise "MiqSshUtil::exec - Command #{cmd}, exited with status #{status}" if errBuf.empty?
-            raise "MiqSshUtil::exec - Command #{cmd} failed: #{errBuf}, status: #{status}"
+          if !signal.nil? || status.nonzero? || !errBuf.empty?
+            raise "MiqSshUtil::exec - Command '#{cmd}', exited with signal #{signal}" unless signal.nil?
+            raise "MiqSshUtil::exec - Command '#{cmd}', exited with status #{status}" if errBuf.empty?
+            raise "MiqSshUtil::exec - Command '#{cmd}' failed: #{errBuf.chomp}, status: #{status}"
           end
           return outBuf
         end
 
         $log.debug "MiqSshUtil::exec - Command: #{cmd} started." if $log
         channel.exec(cmd) do |chan, success|
           raise "MiqSshUtil::exec - Could not execute command #{cmd}" unless success
-          if stdin.present?
+          unless stdin.nil?
```

Comment:
Why change this?  The present? was clearer than the new double negative, IMO.

Reply:
If you're dropping activesupport, then perhaps, just `if stdin`?

Reply:
Alright, I'll have to add an explicit require line for require 'active_support/core_ext/object/blank', though.

Reply:
@Fryguy Updated. :)